src/Tgstation.Server.Host/Components/Session/SessionControllerFactory.cs

```diff
@@ -95,6 +97,11 @@ sealed class SessionControllerFactory : ISessionControllerFactory
 		/// </summary>
 		readonly Api.Models.Instance instance;
 
+		/// <summary>
+		/// The <see cref="IEventConsumer"/> for the <see cref="SessionControllerFactory"/>
+		/// </summary>
+		private IEventConsumer EventConsumer { get; }
```

Comment:
Make it a readonly field, put it with the other interfaces, above the logger factory